### PR TITLE
Updates to github actions to support slack notifs and gitflow branching

### PR DESCRIPTION
**Pull request by estohlmann**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/code.merge.main-to-develop.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v4
+        with:
+          ref: main
+      - name: merge main into develop
+        run: |
+          git config --global user.email "petermul@amazon.com"
+          git config --global user.name "github_actions_mlspace"
```
Comment (petermuller) [inline .github/workflows/code.merge.main-to-develop.yml:22]: MLSpace reference

File: .github/workflows/code.release.branch.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Develop Branch
+        uses: actions/checkout@v4
+        with:
+          ref: develop
+      - name: Create Release Branch and Update Version
+        run: |
+          git config --global user.email "petermul@amazon.com"
+          git config --global user.name "github_actions_mlspace"
```
Comment (petermuller) [inline .github/workflows/code.release.branch.yml:25]: MLSpace reference

**Review by petermuller — CHANGES_REQUESTED** [review]
I think it looks ok once we update the MLSpace -> LISA references

File: .github/workflows/code.release.branch.yml (hunk 2 above)
Comment (estohlmann) [reply .github/workflows/code.release.branch.yml]: Updated

File: .github/workflows/code.merge.main-to-develop.yml (hunk 1 above)
Comment (estohlmann) [reply .github/workflows/code.merge.main-to-develop.yml]: Updated

**Pull request merged**